### PR TITLE
PR

### PR DESCRIPTION
**Pull request by Kethex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
 # lap2_assignment
```
[the remark this answers is not in the document]
Comment (AjaySingh4) [reply README.md:1]: Could change the layout to look more like the one Beth said to base it off - 'Telegraph'. Also getting the incomplete code finished and eliminate the errors that show.

Comment (Deb0890) [inline README.md:1]: Really nice clear instructions in the readme

File: client/requests.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ async function postEntry(e) {
     };
 
     console.log(options);
-    const response = await fetch('http://localhost:3000/routes/entries', options); //THIS IS THE ERROR CAUSING PART (the first parameter in fetch), THOUGH LINE 24 DOES NOT LOG TO THE console either.
-    const result = await response.json(); //This was unfinished - I would use result.id if I could, to get the id. If it doesn't work, I'd probably have to add "this.id = data.id" to the first line under the constructor in models.
```
Comment (Deb0890) [inline client/requests.js:30]: Thanks for the comment explaining the error and what you would do to improve the code

File: client/style.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+body {
```
Comment (Deb0890) [inline client/style.css:1]: Stuck to the brief with a really simple layout

**Review by Deb0890 — COMMENTED** [review]
Overall really good file structuring using mvc, as well as with the client side script files. Kept it simple with post and get requests. Found your code really easy to read through and understand your thinking process. Good stuff guys!

**Pull request closed without merging**